pkg/client/status_reporter.go

```diff
@@ -112,6 +114,7 @@ func (r *StatusReporter) SetFailed(statusErr error) error {
 	conditions.setCondition(v1.OperatorAvailable, v1.ConditionFalse, "", time)
 	conditions.setCondition(v1.OperatorProgressing, v1.ConditionFalse, "", time)
 	conditions.setCondition(v1.OperatorDegraded, v1.ConditionTrue, fmt.Sprintf("Failed to rollout the stack. Error: %v", statusErr), time)
+	conditions.setCondition(v1.OperatorUpgradeable, v1.ConditionFalse, "", time)
```

Comment:
Same as above, we should set the msg here to why this is not upgradable currently.

Reply:
fixed